Quaver.Shared/Screens/Select/UI/Modifiers/DrawableModifier.cs

```diff
@@ -115,6 +118,26 @@ private void CreateModifierDescription() => ModifierDescription = new SpriteText
             UsePreviousSpriteBatchOptions = true
         };
 
+        private void RankedStatusIcon()
+        {
+            if (!Modifier.Ranked)
+            {
+                Modifier.UnrankedSprite = new Sprite()
```

Comment:
@JaanJah 

This is good, but the x position isn't centered because the x position is hardcoded. To center it, you can do the following:

```cs
Modifier.UnrankedSprite = new Sprite()
{
    Parent = this,
    Alignment = Alignment.MidLeft,
    Y = ModifierName.Height - 20,
    X = ModifierName.X / 2f,
    UsePreviousSpriteBatchOptions = true,
    Height = 30,
    Width = 30,
    Image = UserInterface.NotificationWarning
};

Modifier.UnrankedSprite.X -= Modifier.UnrankedSprite.X / 2f;
```

It'll give the following result:

![img](https://i.imgur.com/5qkY5IW.png)